osu.Game/Screens/Edit/Compose/Components/EditorBlueprintContainer.cs

```diff
@@ -144,6 +149,15 @@ protected override void SelectAll()
             SelectedItems.AddRange(Beatmap.HitObjects.Except(SelectedItems).ToArray());
         }
 
+        protected void SetHitObjectsAlive(object sender, NotifyCollectionChangedEventArgs e)
```

Comment:
- doesn't need to be `protected`
- probably needs better name (`KeepHitObjectsAlive`), I'd drop the `sender` arg too

Reply:
1. `private` instead i assume?
2. renamed
3. wouldn't dropping `sender` here cause problems in event un/subscription or do I not need to add this to `Dispose()` to begin with? (`Beatmap.SelectedHitObjects.CollectionChanged += / -=` at line 53 & 183)

Reply:
> 1. `private` instead i assume?

Correct.

> 3. wouldn't dropping `sender` here cause problems in event un/subscription or do I not need to add this to `Dispose()` to begin with? (`Beatmap.SelectedHitObjects.CollectionChanged += / -=` at line 53 & 183)

Fair point, can keep as is, but in that case the method should just be named `selectionChanged` or something.